docs/framework/deployment/windows/8.md

```diff
@@ -0,0 +1,17 @@
+# Installing the .NET Framework on Windows 8
+
+You may need the .NET Framework to run an application on Windows 8. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+
+## .NET Framework 4.6.2
+
+You can install the [.NET Framework 4.6.2](https://go.microsoft.com/fwlink/?linkid=845529&source=dotnetdocs) or later on Windows 8. 
+
+The .NET Framework 4.6.2 is included with the latest version of [Windows 10](https://www.microsoft.com/software-download/windows10).
+
+## .NET Framework 3.5
+
+You can install the [.NET Framework 3.5 on Windows 8](installing-dotnet-35-windows-10.md). The instructions are for Windows 10 but can also be used for Windows 8.
```

Comment:
this usually confuses customers. we got similar feedback when we linked from a windows section to a macos article that instructions worked as well. the topic about 3.5 should apply to 8, 8.1 and 10 and then you don't have to say this.